README.md

```diff
@@ -60,6 +60,17 @@ Amplication Data Service Generator is the component that generates the server-si
 
 To use this package as a library or as a CLI with other projects, follow the instructions on the package page.
 
+### [Amplication Design System](./packages/amplication-data-service-generator/README.md)
+
+Amplication Data Service Generator is the component that generates the server-side code (models, Prisma client, REST API, authentication, and authorization filters) of the application built with Amplication. This package is used internally by Amplication server.
+
+To use this package as a library or as a CLI with other projects, follow the instructions on the package page.
+
+### [Amplication Container Builder](./packages/amplication-container-builder/README.md)
```

Comment:
I think this is not "cooked" enough to be used as a separate package and maybe should be removed from the main README for now

Reply:
It works perfectly, I don't see why not

Reply:
But I don't have a strong opinion about it.

Reply:
I suggest keeping the readme simple for the current time